demo/DemoContent.js

```diff
@@ -37,6 +37,7 @@ export class DemoContent extends Component {
         }
 
         <button id="logout" onClick={ () => brandwatchAuthLogout() }>Logout</button>
+        <button id="error" onClick={ () => brandwatchAuthHandleError() }>Error</button>
```

Comment:
Shouldn't we use data-ut-ids here?

Reply:
yes and maybe no, this is just a demo component and so not used externally. I'm not sure benefits outweigh the extra verbosity needed for the selectors.